types/react-dates/tslint.json

```diff
@@ -17,6 +17,7 @@
         "semicolon": false,
         "trim-file": false,
         "typedef-whitespace": false,
-        "whitespace": false
```

Comment:
Why was the new lint exception required?

Reply:
It looks like a new rule was added globally which doesn't like how the namespace is exported as a whole.